tdesign/src/Layout/BaseLayout/BaseLayout.tsx

```diff
@@ -123,6 +123,8 @@ const BaseLayout = ({
             <UserThemeContext.Consumer>
               {({ userColors, setUserColors, toggleDarkMode }) => (
                 <div>
+                  {/* TODO: fix me */}
+                  {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events */}
```

Comment:
(note this is a hidden toggle atm anyway)

Reply:
That makes me feel more calm about not fixing it right away :smile: